docs/pages/index.mdx

```diff
@@ -1,175 +1,220 @@
+videoBanner: BJWbSqiDLeU
+tocDepth: 3
 ---
 
 Teleport is an identity-aware, multi-protocol access proxy. Teleport understands
 the SSH, HTTPS, RDP, Kubernetes API, MySQL, MongoDB and PostgreSQL wire
-protocols, plus many others.
+protocols, plus many others. It can integrate with Single Sign-On providers and
+enables you to apply access policies using infrastructure-as-code and GitOps
+tools.
 
-With Teleport you can:
+See how Teleport works by completing the tutorial below. This shows you how to
+spin up a single-instance Teleport cluster on a Linux server using Teleport
+Community Edition. Once you deploy the cluster, you can configure RBAC, register
+resources, and protect your small-scale demo environments or home lab.
 
-- Replace a mix of vaults, passwords, API keys and tokens with short-lived SSH and X.509 certs.
-- Have a single access point and Single-Sign-On provider for all of your
-  infrastructure, including SSH servers, Kubernetes clusters, databases,
-  desktops, web applications, and more.
-- Write policies with Terraform or Kubernetes resources for all clouds,
-  environments and protocols and manage them with GitOps.
-- Record SSH and `kubectl exec` sessions, DB queries, Windows desktop sessions,
-  web sessions and API requests.
-- Use mutual TLS tunnels to protect your infrastructure endpoints.
+You can also get started right away with a production-ready Teleport cluster by
+signing up for a [free trial of Teleport
+Team](./choose-an-edition/teleport-team.mdx).
 
-<Notice type="info">
+## Set up a demo cluster
 
-If your organization is already using Teleport and you want to learn how to
-access infrastructure, read our [Connect your
-Client](./connect-your-client/introduction.mdx) guides for instructions.
+<Figure width="700">
+![Architecture of the setup you will complete in this
+guide](../img/linux-server-diagram.png)
+</Figure>
+
+We will run the following Teleport services:
+
+- **Teleport Auth Service:** The certificate authority for your cluster. It
+  issues certificates and conducts authentication challenges. The Auth Service
+  is typically inaccessible outside your private network.
+- **Teleport Proxy Service:** The cluster frontend, which handles user requests,
+  forwards user credentials to the Auth Service, and communicates with Teleport
+  instances that enable access to specific resources in your infrastructure.
+- **Teleport SSH Service:** An SSH server implementation that takes advantage of
+  Teleport's short-lived certificates, sophisticated RBAC, session recording,
+  and other features.
+
+### Prerequisites
+
+<Notice type="tip">
+
+You will need the following to deploy a demo Teleport cluster. If your
+environment doesn't meet the prerequisites above, you can get started with
+Teleport by signing up for a [free trial of Teleport Enterprise
+Cloud](https://goteleport.com/signup/).
 
 </Notice>
 
-## Try out Teleport
+- A Linux host with only port `443` open to ingress traffic. You must be able
```

Comment:
Does LE no longer require port `80` to complete its validation prior to issuing a cert?

Reply:
Teleport uses the TLS-ALPN-01 challenge, which doesn't require port 80